lib/appium_lib/common/helper.rb

```diff
@@ -108,7 +108,7 @@ def get_page_class
     # Count all classes on screen and print to stdout.
     # Useful for appium_console.
     def page_class
-      puts get_page_class
+      Appium::Logger.info get_page_class
```

Comment:
This exists only to output to stdout, if you want to use it without messages then invoke get_page_class.